packages/fx-core/src/plugins/solution/fx-solution/debug/provisionLocal.ts

```diff
@@ -129,3 +140,136 @@ export async function setupLocalDebugSettings(
   TelemetryUtils.sendSuccessEvent(TelemetryEventName.setupLocalDebugSettings, telemetryProperties);
   return ok(Void);
 }
+
+export async function configLocalDebugSettings(
+  ctx: v2.Context,
+  inputs: Inputs,
+  localSettings: Json
+): Promise<Result<Void, FxError>> {
+  const includeFrontend = ContextHelper.includeFrontend(ctx);
+  const includeBackend = ContextHelper.includeBackend(ctx);
+  const includeBot = ContextHelper.includeBot(ctx);
+  const includeAuth = ContextHelper.includeAuth(ctx);
+  let trustDevCert = localSettings?.frontend?.trustDevCert as boolean | undefined;
+
+  const telemetryProperties = {
+    platform: inputs.platform as string,
+    frontend: includeFrontend ? "true" : "false",
+    function: includeBackend ? "true" : "false",
+    bot: includeBot ? "true" : "false",
+    auth: includeAuth ? "true" : "false",
+    "trust-development-certificate": trustDevCert + "",
+  };
+  TelemetryUtils.init(ctx);
+  TelemetryUtils.sendStartEvent(TelemetryEventName.configLocalDebugSettings, telemetryProperties);
+
+  try {
+    if (inputs.platform === Platform.VSCode || inputs.platform === Platform.CLI) {
+      const isMigrateFromV1 = ContextHelper.isMigrateFromV1(ctx);
+
+      const localEnvMultiProvider = new LocalEnvProvider(inputs.projectPath!);
```

Comment:
```Suggestion
      const localEnvProvider = new LocalEnvProvider(inputs.projectPath!);
```
Since you're already renamed the type  name, also rename the variable consistently.

Reply:
Thanks! Renamed them